pkgs/development/python-modules/rauth/default.nix

```diff
@@ -0,0 +1,61 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, fetchpatch
+, requests
+, pytestCheckHook
+, mock
+, coverage
+, flake8
+, nose
+, pycrypto
+, yanc
+,
+}:
+buildPythonPackage rec {
```

Comment:
```suggestion
}:

buildPythonPackage rec {
```

Reply:
Addressed in [force-push](https://github.com/NixOS/nixpkgs/compare/bb60250fa9142e184d9893b23520adb1d0cc01fe..27344c890fe0e4462daaf9959ca22aede8f4c771)